data-manipulation/prng/get-generation-random-numbers.yml

```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: get generation random numbers
+    namespace: data-manipulation/prng
+    author:
+      - michael.hunhoff@fireeye.com
+      - johnk3r
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
```

Comment:
Random numbers can be used for a variety of tasks, so I suggest to remove this here.

Reply:
Got it.